src/App.js

```diff
@@ -2,10 +2,10 @@ import './App.css';
 import { Routes, Route } from 'react-router-dom';
 import { useEffect } from 'react';
 import { useDispatch } from 'react-redux';
-import Header from './components/header/Header';
-import Missions from './components/mission/Missions';
 import Rocket from './components/rockets/Rocket';
 import Profile from './components/profile/Profile';
+import Header from './components/header/Header';
+import Missions from './components/mission/Missions';
```

Comment:
- [ ] As mentioned by the previous reviewers you should fix the errors that come when running the project. The directory `components` doesn’t exist in your project but is `Components` with `C`, is in uppercase and not lowercase that why you get the errors. Kindly fix that 🙃 🙃 

<img width="1121" alt="Screenshot 2023-04-21 at 19 54 14" src="https://user-images.githubusercontent.com/74501523/233695060-903c8364-d77a-4993-ae66-fc2d8e52312d.png">